solarwinds_apm/configurator.py

```diff
@@ -157,6 +157,18 @@ def _configure_otel_components(
                     apm_txname_manager,
                     apm_config,
                 )
+                # While in legacy mode, user can also opt into exporting
+                # Otel instrumentor metrics and logs by OTLP.
+                # Default values are set by SolarWindsDistro; user can customize.
+                if apm_config.get("export_metrics_enabled") is True:
+                    self._configure_metrics_exporter(apm_config)
+                    self._configure_otlp_metrics_span_processors(
```

Comment:
In legacy mode i assume the response time metric (`trace.service.response_time`) is being calculated by our `SolarWindsInboundMetricsSpanProcessor` and exported via apm-proto. In this case, do we need to `_configure_otlp_metrics_span_processors` since that is the OTLP counterpart to processing spans for response time?

Reply:
Yes, you're right! Removed in [27204c4](https://github.com/solarwinds/apm-python/pull/508/commits/27204c4c224c7aefb1092cbcf7ffa21b97d5ffa8) and updated in-code comments. Also saw an opportunity to simplify the helper in [20eb231](https://github.com/solarwinds/apm-python/pull/508/commits/20eb2319c78e3a4c81a74f158a433e531c241b7d) as exporters always init first anyway for the non-legacy case.